src/app/containers/manuscript/manuscript-toc.tsx

```diff
@@ -15,7 +15,7 @@ export const ManuscriptTOC: React.FC<ManuscriptTOCProps> = (props) => {
     <React.Fragment>
       <section className={classes.toolbarPlaceholder} aria-hidden="true">
         <h1 className={classes.title}>Libero Editor</h1>
-        <h3 className={classes.version}>v0.0.1</h3>
+        <h3 className={classes.version}>{`v${process.env.REACT_APP_VERSION}`}</h3>
```

Comment:
I think we should mock this somehow for testing. Or we'll end up with a lot of updates for the snapshot

Reply:
Yeah, good call. I'll take a look and see what I can do.